### PR TITLE
Add an openshift test

### PR DESCRIPTION
**Pull request by stefwalter**
This adds an openshift test image too.

### CODE REVIEW

**Comment by petervo** [comment]
Do we need to skip this test on rhel like we do the other kube tests?

**Comment by stefwalter** [comment]
> Do we need to skip this test on rhel like we do the other kube tests?

Done.

File: test/check-openshift [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        self.openshift.wait_boot()
+
+        MachineCase.setUp(self)
+
+        tmpfile = os.path.join(self.tmpdir, "config")
+        self.openshift.download("/root/.kube/config", tmpfile)
+
+        m = self.machine
+        with open(tmpfile, "r") as f:
+            m.execute("mkdir -p /home/admin/.kube && cat > /home/admin/.kube/config", input=f.read())
```
Comment (mvollmer) [inline test/check-openshift]: Isn't this just `mkdir -p /home/admin/.kube && cp /root/.kube/config /home/admin.kube`?

Comment (mvollmer) [inline test/check-openshift]: Heh, different machines, sorry.

**Pull request closed without merging**